deployments/examples/ocis_full/inbucket.yml

```diff
@@ -8,6 +8,8 @@ services:
       NOTIFICATIONS_SMTP_USERNAME: notifications@${OCIS_DOMAIN:-ocis.owncloud.test}
       # the mail catcher uses self signed certificates
       NOTIFICATIONS_SMTP_INSECURE: "true"
+      # enable the notifications service as it is not started automatically
+      OCIS_ADD_RUN_SERVICES: "notifications"
```

Comment:
That is imho not necessary.
We have in `ocis.yml` the `OCIS_ADD_RUN_SERVICES: "notifications"` command present. As the yml files are combined together, in particular we merge the ocis sections, this makes no sense. In addition it would mean that we have to maintain two sources.

Reply:
yes. I deleted from `OCIS_ADD_RUN_SERVICES: "notifications"` from ocis.yml and put here. Since Michael suggests not removing it from ocis.yml -> of course I will remove it from here